doc/setup_instructions.md

```diff
@@ -79,29 +94,19 @@ https://github.com/godotengine/godot-monodevelop-addin This is not
 needed for basic usage, so you can skip if you can't figure out how to
 install it.
 
-On Windows you should be able to install Visual Studio 2019 with the
-desktop development featureset, that should have everything needed for
-C# development. Please be patient with this as this is going to take
-tens of minutes to install and take a lot of disk space.
+### Visual Studio 2019
 
-It might also be possible to make things work with Visual Studio Code
-as well, but there are no instructions for that. Note: that Godot
-requirements say that Visual Studio build tools are enough but they
-may not include nuget, which is needed, so if you go that route you
-may need to manually install nuget.
+On Windows you can use Visual Studio 2019 to work on Thrive. You can
+find download and setup instructions here https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_basics.html#configuring-vs-2019-for-debugging
 
 ### Visual Studio Code
 
-To setup Visual Studio Code  work with Godot, you'll need
-Visual Studio Code. You can install Visual Studio Code from here:
-https://code.visualstudio.com/
```

Comment:
Shouldn't the mention from where to download visual studio code kept?

Reply:
It seems kind of pointless when you can just search Visual Studio Code online and find the site.

Reply:
I think it reduces possible confusion between visual studio proper and visual studio code, if the link is here.

Reply:
How about a quick explanation on what makes them different? Going over that they are different programs, and that Visual Studio is a full fledged IDE and VSCode is a text editor and the main differences.

Reply:
That's not in scope of these instructions, a single sentence saying "not to be confused with visual studio" would be enough.